### ISSUE TITLE
Please add "Most popular tags" to sidebar

### ISSUE THREAD

**Issue by dschwabeS11** [Perlkonig/grav-theme-knowledge-base#25]
It would be awesome if you could add "most popular tags" to the sidebar, then we don't have to build this on our own and others can use this feature as well.

Tags should rather be separated by commas or semicolons then be in a list. This way more tags fit without taking to much space.

Thanks!

**Comment by Perlkonig** [comment]
[There's a plugin for that.](https://github.com/Perlkonig/grav-plugin-tagcloud) I guess I could add that to the default skeleton, if people would find that useful. I'll add it to my TODO. Leaving the issue open to remind me.

**Comment by ghost** [comment]
Thanks. I installed and integrated it. 
There is an [issue](https://github.com/Perlkonig/grav-plugin-tagcloud/issues/2) though.

For me it makes sense to integrate this into the default skeleton and as theme dependency.

**Comment by Perlkonig** [comment]
The more I tinker with this, the more I think I'd prefer just a straight list of the top X tags, like I've done with the other groups. That was your original ask anyway.

**Comment by ghost** [comment]
Yes it even still is. - For now we integrated the tagcloud but are not sure if we want to keep it.
You could even integrate both and make them appear and disappear with options like the sidebar elements do right now. 
What do you think?

**Comment by Perlkonig** [comment]
This is a functionality best separated from the theme itself. I'm transferring this to the skeleton repo. Thanks!
